osutil/user.go

```diff
@@ -28,6 +28,8 @@ import (
 	"regexp"
 	"strconv"
 	"strings"
+
+	"github.com/snapcore/snapd/release"
```

Comment:
Personally, just for import hygiene I would do it the other way around. Have this module just take an argument and have the caller import release and decide how to proceed.